tests/apis.js

```diff
@@ -0,0 +1,430 @@
+import test from 'ava';
+import uuid from 'uuid';
+import ipsum from 'lorem-ipsum';
+import Promise from 'bluebird';
+import slugify from 'underscore.string/slugify';
```

Comment:
We talked about it, going to use `underscore.string` for string manipluation instead of lodash